packages/RatPack.package/RPModel.class/class/createJsonObject..st

```diff
@@ -0,0 +1,7 @@
+object conversion
+createJsonObject: aJsonString
+
+	(self validateJson: aJsonString) 
+		ifFalse: [self notify: 'Invalid Json! Could not create Object on ' , self name].
```

Comment:
This condition forces that aJsonString must always include all possible attributes, see [validateJson.st](https://github.com/hpi-swa-teaching/RatPack2.0/blob/develop/packages/RatPack.package/RPModel.class/class/validateJson..st). Is this what we want? I don't think so.

Reply:
Actually it does not force to have all keys in your json, it only forces to not have any other than the allowed and to give attributes where no default value exists.

in [validateJson.st](https://github.com/hpi-swa-teaching/RatPack2.0/blob/develop/packages/RatPack.package/RPModel.class/class/validateJson..st) they make a Set operation and combine with given + default value keys with all attributes.

So you are partly right, and we should change it, that all the keys from the given json exist in the list of the others but not the other way around.

Reply:
Do you think for the other purposes like post this validateJson Method is correct ?
becasue for the post we need this enforcement right ? So I have to create a new validate Json method only for the put ?!

Reply:
yes.